pkg/ui/react-app/yarn.lock

```diff
@@ -6081,7 +6081,7 @@ hoist-non-react-statics@^3.3.1:
   dependencies:
     react-is "^16.7.0"
 
-hosted-git-info@^2.1.4:
+hosted-git-info@2.8.9, hosted-git-info@^2.1.4:
   version "2.8.9"
```

Comment:
Looks like we are already using version `2.8.9`, maybe we can just bump the minimum allowed version (^2.1.4 to ^2.8.9)

 ```suggestion
hosted-git-info@^2.8.9:
  version "2.8.9"
```

Reply:
I guess we don't need this change. As you said, we already have the correct version as a dependency.